css/style.css

```diff
@@ -247,14 +253,117 @@ nav a {
     max-height: 422px;
 }
 
+/* -- RESPONSIVE HEADER AND SERACH BAR -- */
```

Comment:
minor typo -- only mentioning because instructor / others who view our code in portfolio, etc.